lib/player.js

```diff
@@ -6687,7 +6690,11 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
             variant.allowedByKeySystem = true;
 
             for (const keyId of stream.keyIds) {
-              const keyStatus = keyStatusMap[isGlobalStatus ? '00' : keyId];
+              let keyStatus = keyStatusMap[isGlobalStatus ? '00' : keyId];
+              if (!isGlobalStatus && !keyStatus && keyId.length == 22) {
```

Comment:
Same here.  I really don't think this should be duplicated everywhere we read key statuses.  The key IDs should be in a consistent format.

Reply:
Reverted